core/views.py

```diff
@@ -40,6 +46,7 @@ def get_context_data(self: "PostDetailView", **kwargs: dict[str, Any]) -> dict[s
         context = super().get_context_data(**kwargs)
         comments = self.object.get_comments()
 
+        context["default_avatar_url"] = settings.DEFAULT_AVATAR_URL
```

Comment:
Use default on author object in model or as described above - author.get_avatar function.

Reply:
Okey, I have already done it, it will be fixed after PR